system-addon/common/Actions.jsm

```diff
@@ -110,6 +110,15 @@ for (const type of [
   actionTypes[type] = type;
 }
 
+const ASRouterActions = {};
+for (const type of [
```

Comment:
If this is intended to be a whitelist of actions available to external message providers, I think a comment describing at such here would be good